aea/runtime.py

```diff
@@ -258,7 +258,11 @@ def _start(self) -> None:
 
         self._agent.multiplexer.set_loop(asyncio.new_event_loop())
 
-        self._agent.multiplexer.connect()
+        # it is important that 'connect' is before 'start setup'.
```

Comment:
This comment should be added in the connect docstring as well.

Reply:
Maybe It would be better to call `connect` in `start_setup`. This would make usage more straightforward.